guides/common/modules/ref_system-requirements.adoc

```diff
@@ -3,105 +3,61 @@
 [id="system-requirements_{context}"]
 = System requirements
 
-The following requirements apply to the networked base operating system:
+ifdef::satellite[]
+{ProductName} is fully supported on both physical systems and virtual machines that run on hypervisors that are supported to run {RHEL}.
+For more information about certified hypervisors, see https://access.redhat.com/articles/973163[Certified Guest Operating Systems in Red Hat OpenStack Platform, Red Hat Virtualization, Red Hat OpenShift Virtualization and Red Hat Enterprise Linux with KVM].
```

Comment:
The KCS is hosted at https://access.redhat.com/articles/certified-hypervisors these days, should we update the link to avoid redirecots?